harvard-huit/Harvard-Geospatial-Library
# Index map legend, icon tooltips, canned search, constraints bug, code commenting cleanup

**Index map legend, icon tooltips, canned search, constraints bug, code commenting cleanup**
* * *

# What does this Pull Request do?
FE enhancements and bug fixes.

# How should this be tested?
Index map legend
* A legend should appear above index maps ([index map example](https://localhost:3001/catalog/stanford-bj754sp5315)), but not appear above any other type of map ([non-index map example](https://localhost:3001/catalog/stanford-jh879ks6412) )

Svg icon tooltip labels
* Initially used javascript to include labels next to their icons, but discovered a way to do this without JS
* The institution name is also shortened, so it'll say "MIT" and "Harvard" instead of "Massachusetts Institute of Technology" and "Harvard University"
* Confirm that labels appear next to icons on both [search results](https://localhost:3001/?utf8=%E2%9C%93&search_field=all_fields&q=) and [item detail](https://localhost:3001/catalog/mit-f6rqs4ucovjk2)  pages

Canned searches
* Updated [homepage](https://localhost:3001/) to include links to the desired canned searches (the 4 cards on the page)
* Confirm that links work (note that 'population data' one may still break, but is being worked on)

BBox search constraint  bug fix
* Fixed bug where bounding box constraint was not appearing on the search results page after moving around on the map (but was appearing in the url string)
* To test, move around on map and the bbox constraint should appear next to the search box with any other selected filters

Code commenting
* Other files changed in this PR are updates to the type of commenting used in the html.erb files (switched from `` to `<%# %>` so that the comments no longer appear in the inspector)


# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? no
- integration tests? no

# Interested parties
@phil-plencner-hl 
@enriquediaz
